src/View/Advertisement/_carousel.html.twig

```diff
@@ -0,0 +1,31 @@
+<div id="carouselExampleControls" class="carousel slide col-12 col-sm-9 p-0" data-ride="carousel">
+  <div class="carousel-inner">
+    {% block carouselImages %}
+      {% if photos is empty %}
```

Comment:
on touche au but
en Twig, tu peux utiliser la syntaxe
for / else  (ça t'évite de faire ce if)

Reply:
ah merci pour le "for / else" :)